packages/shared/src/lib.ts

```diff
@@ -1,6 +1,22 @@
-export const USER_EVENTS = {
+export const UserEvents = {
   UsernameCreated: 'username-created',
   UsernameMissing: 'username-missing',
 } as const;
+export type UserEvents = typeof UserEvents[keyof typeof UserEvents];
 
-export type UserEvents = typeof USER_EVENTS[keyof typeof USER_EVENTS];
+export const GameState = {
```

Comment:
Should we not be consistent with singular/plural? Either `UserEvents`/`RoomEvents`/`GameStates` or `UserEvent`/`RoomEvent`/`GameState`.

Reply:
Yes agree I think it should be singular though, thinking at enums again. But I also don't have a strong opinion on this, so happy with what you like

Reply:
Yeah, fine, let's make it singular.